src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -29,19 +26,16 @@ public Command parse(String args) {
                 new ArgumentTokenizer(PREFIX_PRIORITY, PREFIX_STATUS, PREFIX_NOTE,
                         PREFIX_STARTTIME, PREFIX_ENDTIME, PREFIX_TAG);
         argsTokenizer.tokenize(args);
-        argsTokenizer.createFloatingTask();
         try {
             return new AddCommand(
                     argsTokenizer.getPreamble().get(),
-                    argsTokenizer.getValue(PREFIX_PRIORITY).get(),
-                    argsTokenizer.getValue(PREFIX_STATUS).get(),
-                    argsTokenizer.getValue(PREFIX_NOTE).get(),
-                    argsTokenizer.getValue(PREFIX_STARTTIME).get(),
-                    argsTokenizer.getValue(PREFIX_ENDTIME).get(),
+                    argsTokenizer.getValue(PREFIX_PRIORITY).orElse("none"),
+                    argsTokenizer.getValue(PREFIX_STATUS).orElse("incomplete"),
+                    argsTokenizer.getValue(PREFIX_NOTE).orElse(""),
+                    argsTokenizer.getValue(PREFIX_STARTTIME).orElse("31/12/2017 00:00"),
+                    argsTokenizer.getValue(PREFIX_ENDTIME).orElse("31/12/2017 23:59"),
                     ParserUtil.toSet(argsTokenizer.getAllValues(PREFIX_TAG))
             );
```

Comment:
To properly handle floating task, orElse() should pass in empty Optionals instead of having default values for Priority, Note and Deadline